.github/workflows/deploy.yml

```diff
@@ -0,0 +1,53 @@
+name: Deploy
+
+on:
+  push:
+    branches:
+      - develop
+      - master
+
+jobs:
+  canary:
+    runs-on: ubuntu-latest
+    if: github.ref_name == 'develop'
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 18
+      - uses: actions/cache@v3
```

Comment:
Did you mean to leave these two using `actions/cache` instead of `setup-node` caching + `npm ci`?